src/mtcp/Makefile.in

```diff
@@ -5,9 +5,6 @@ MANA_USE_LH_FIXED_ADDRESS = @MANA_USE_LH_FIXED_ADDRESS@
 CC = @CC@
 M32=@M32@
 CFLAGS = @CFLAGS@
-ifeq ($(findstring cori,$(PLATFORM)),cori)
-  CFLAGS += -DGNI
-endif
```

Comment:
why is this removed?

Reply:
I cannot find anywhere using macro `GNI`, remove for simplicity.